app/src/main/java/com/vyvu/kalculatorapp/Calculator.kt

```diff
@@ -0,0 +1,109 @@
+package com.vyvu.kalculatorapp
+
+class Calculator {
+    var result = 0.0
+    var infix = mutableListOf<String>()
+    private var postfix = mutableListOf<String>()
+
+    fun calc(): Double {
+        initPostfix()
+        val stack = Stack()
+        for (i in postfix) {
+            val num = i.toDoubleOrNull()
+            if (num != null) stack.push(num)
```

Comment:
```suggestion
            if (num != null) {
                stack.push(num)
            }
```
if else mà trên 1 dòng mới bỏ dấu ngoặc nhọn nhé